drivers/bridge/bridge.go

```diff
@@ -431,10 +428,21 @@ func (d *driver) Config(option map[string]interface{}) error {
 		d.config = config
 	} else {
 		config = &configuration{}
+		d.config = config
 	}
 
 	if config.EnableIPForwarding {
-		return setupIPForwarding(config)
+		err := setupIPForwarding(config)
```

Comment:
In original code, we would return here without storing err and whether there is an error or not. Now, we only return if there is an error. Otherwise, we continue to process the EnableIPTables option.